MaxText/maxengine_server.py

```diff
@@ -17,6 +17,7 @@
 import jax
 import os
 import sys
+import pathwaysutils  # pylint: disable=unused-import
```

Comment:
@lukebaumann FYI these imports are being removed in [PR 1482](https://github.com/AI-Hypercomputer/maxtext/pull/1482). Let us know if there are any issues with that